specification/2.0/README.md

```diff
@@ -178,6 +178,10 @@ To simplify client-side implementation, glTF has additional restrictions on JSON
 
    > **Implementation Note:** This allows generic glTF client implementations to not have full Unicode support. Application-specific strings (e.g., values of `"name"` properties or content of `extras` fields) may use any symbols.
 3. Names (keys) within JSON objects must be unique, i.e., duplicate keys aren't allowed.
+4. Numbers defined as integers in the schema must be written without fractional part (i.e., `.0`). Exporters should not produce integer values greater than 2<sup>53</sup> because some client implementations will not be able to read them correctly.
```

Comment:
What is the purpose of prohibiting a fractional part? It may be parsed differently in a typed language, I suppose? If any mainstream JSON serializer doesn't do this, it won't be within a developer's ability to change it. If it's already universal in common libraries, as it is for JS, that's fine.

Note that JSON also allows an exponent part: https://stackoverflow.com/a/19554986/1314762. Should that be mentioned?

Reply:
From JSON-Schema spec:
> Some programming languages and parsers use different internal representations for floating point numbers than they do for integers.

> For consistency, integer JSON numbers SHOULD NOT be encoded with a fractional part.

We had this issue with files made by the Blender exporter at one point. `JSON.stringify` is fine. Note, that this affects only properties marked as "integer" in the schema such as:
```json
"buffers": [
  {
    "byteLength": 25.0
  }
]
```
I'm fine with changing "must" to "should" to align better with JSON-Schema language. Exponent notation is usually associated with floats, so it shouldn't be used for integers as well.

Reply:
It might only be remotely related, but related: There has been some discussion about this at https://github.com/KhronosGroup/glTF-Validator/issues/8

Reply:
I think that loaders using typed languages that rely on "integer" schema type are a bigger concern here rather than half-baked serializers. For example, `"componentType":5123.0` crashes some loaders.

Reply:
Ok, i'm fine with either "must" or "should", whichever you prefer. Let's also mention exponents here.